R/init.R

```diff
@@ -21,12 +21,16 @@
 .pkgenv <- new.env(parent=emptyenv())
 
 .onAttach <- function(libname, pkgname) {
+    if (getOption("blpVerbose", FALSE)) {
```

Comment:
I am thinking we could even make this unconditional, as users can still suppress startup messages via `suppressMessages()` and `suppressPackageStartupMessages()`.